### DISCUSSION TITLE
AutoEvents not sent to core-data

### DISCUSSION THREAD

**Discussion by johker** [edgexfoundry/community#23] [Q&A]
Hi everyone, 

I'm running the ireland release (with security) and I'm using the modbus device service with a configured AutoEvent and I can see in the logs that it is receiving data:
![image](https://user-images.githubusercontent.com/4265783/198222712-9fcfb869-4a05-406c-9a15-46c248e70562.png)

However no events or readings are listed in the core-data service. Am I missing some additional configuration? 

Thanks in advance
Johannes

[docker-compose-ireland.txt](https://github.com/edgexfoundry/community/files/9876981/docker-compose-ireland.txt)

**Comment by cloudxxx8** [comment]
how do you verify there is no events in core-data?  please write down the whole recreate steps, and the log from core-data might be helpful.
also, we strongly recommend you to use the Jakarta release, because it is the LTS release.
we won't maintain ireland release anymore.

**Reply by johker** [reply]
I switched to jakarta with the same result. I have used the following to check the events:
`curl --location --request GET 'https://10.5.240.130:8080/core-data/api/v2/event/all' \
--header 'Authorization: Bearer eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiI5ODNlOWI4YS03NTQwLTQ2N2UtODhkNS04ZDI5NjBjZmQ5NjciLCJleHAiOjE2NjczMTQ0MzMsIm5iZiI6MTY2NzMxMDgzMywiaWF0IjoxNjY3MzEwODMzfQ.jcZmvDBK-kV1hp-VvFtytEgoXZLIlp-kuAw5ewqGDh2xJuXa8eYRjwE6uTOAz9BXlG4VXzcnRG4k8skzEmh1dg' \
--data-raw ''`

It gives:
`{
    "apiVersion": "v2",
    "statusCode": 200,
    "totalCount": 0,
    "events": []
}`

The core-data logs show nothing unusual (see attachment). I'm using the default configuration.toml of jakarta for device-modbus. 
[core-data.log](https://github.com/edgexfoundry/community/files/9909520/core-data.log)
[configuration.toml.txt](https://github.com/edgexfoundry/community/files/9909555/configuration.toml.txt)

**Reply by cloudxxx8** [reply]
are you running everything from the docker-compose file?
the problem should be between device-modbus and redis (message bus) or core-data and redis.
please write down all your recreste steps
it's probably about docker network settings
